Replaced the @import keyword (#53)
KittyGiraudel/sass-boilerplate: stylesheets/abstracts/README.md

The rule of thumb for this folder is that it should not output a single line of CSS when compiled on its own. These are nothing but Sass helpers.

## \_index.scss

> With the deprecation of `@import` keyword, We now use `@use` and `@forward` keywords. Unlike `@import` keyword, adding abstracts at the top of the main.scss file does not make them globally accessbile to eveything else comes afterwards, when using `@use` keyword.

Reviewer: ```suggestion
With the deprecation of the `@import` keyword, we now use the `@use` and `@forward` keywords. Unlike with `@import`, adding abstracts with `@use` at the top of the `main.scss` file does not make them globally accessible to eveything else coming afterwards.
```

Author: done.

> Because of that we need to import those abstracts using `@use` keyword each time we need them inside the individual files.

Reviewer: ```suggestion
Because of that we need to import those abstracts using `@use` keyword each time we need them inside the individual files, similar to how we would import utilities in the JavaScript world for that matter.
```

Author: done.

This is where we need the `@forward` keyword.

> We can create an `_index.scss` file. In this file, we can collect all the abstract partials with `@forward` keyword and we can re-export all abstract modules when we need them as a singular thing.

Reviewer: ```suggestion
We can create an `_index.scss` file. In this file, we can collect all the abstract partials with the `@forward` keyword and re-export all abstract modules when we need them as a singular entry.
```

Author: done.
